spec/features/tasks/new_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Create a task', type: :feature do
+  include_context 'when user signed in'
+
+  let(:project) { create(:project, name: 'A test project') }
+
+  scenario 'user creates a task' do
+    visit projects_path
+    click_on 'New Project'
+
+    fill_in 'Name', with: 'A test project'
+    fill_in 'Description', with: 'Test description'
+
+    click_button 'Create Project'
```

Comment:
Is it related to creating a task?

Reply:
No, you should delete it.

Reply:
There is a better way to create a project using factory bot, try just `create(:project, name: 'A test project')`